lib/active_model/serializer/adapter/json.rb

```diff
@@ -1,47 +1,128 @@
 class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
-        extend ActiveSupport::Autoload
```

Comment:
(reason is it makes the diff easier to read, and we'll hopefully go back to a more standard class definition when we make Adapter a module and Adapter::Base the superclass